react/features/authentication/middleware.ts

```diff
@@ -289,7 +292,12 @@ function _handleLogin({ dispatch, getState }: IStore) {
                 titleKey: 'dialog.tokenAuthFailedTitle'
             }, NOTIFICATION_TIMEOUT_TYPE.LONG));
 
-            dispatch(redirectToDefaultLocation());
+            if (receivedTokenAuthServiceUrl) {
```

Comment:
Why did you rename this from `tokenAuthServiceUrl` ? It's confusing, since you don't receive the URL from anywhere.

You also want to drop the notification above.

Last, if `isTokenAuthEnabled()` returns `true` we are guaranteed to have a URL, so this check is misleading.

Reply:
Reworked.